metadata-ingestion/src/datahub/ingestion/source/sql/hive_metastore.py

```diff
@@ -822,6 +822,15 @@ def loop_views(
                 aspect=view_properties_aspect,
             ).as_workunit()
 
+            if (
+                dataset.view_definition
+                and self.config.mode in [HiveMetastoreConfigMode.hive]
+                and self.config.include_view_lineage
+            ):
+                self.aggregator.add_view_definition(
+                    view_urn=dataset_urn,
+                    view_definition=dataset.view_definition,
+                )
```

Comment:
hive_metastore source's `get_workunits_internal` does not invoke parent source's method. so we should either remove this change or fix it fully. For simplicity, you can remove this change from here and tests. 

Let's hide `include_view_lineage` config in the sources that do not support view lineage extraction. Possibly also add a note in their docs that we do not currently support view lineage extraction for them. This can be done by setting/editing capability `@capability(SourceCapability.LINEAGE_COARSE, "View lineage is not supported", supported=False)`